packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -124,21 +135,20 @@ const main = () => {
             '-k, --key <api-key>',
             `find your API key at https://runtime.commercecloud.com/account/settings`,
             (val) => {
-                if (!(typeof val === 'string' && val.length > 0)) {
-                    throw new program.InvalidArgumentError(`"${val}" cannot be empty`)
+                if (!(typeof val === 'string') && val.length > 0) {
```

Comment:
```suggestion
                if (typeof val !== 'string' || val === '') {
```
I'm not sure if this change was intentional, but I think that this is what the condition is supposed to be.